logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/keypackage/RefillKeyPackageUseCase.kt

```diff
@@ -51,6 +51,8 @@ class RefillKeyPackagesUseCaseImpl(
 ) : RefillKeyPackagesUseCase {
     override suspend operator fun invoke(): RefillKeyPackagesResult =
         currentClientIdProvider().flatMap { selfClientId ->
+            // TODO: Maybe use MLSKeyPackageCountUseCase instead of repository directly,
+            //       and fetch from local instead of remote
```

Comment:
@typfel comments on this one?

I am not sure much we can/should trust the local MLS client when getting keypackage count.

The way I've plugged it in, it just does it relying on remote info, which can be a bit slow when coming back from a few days of inactivity.